auth-api/pom.xml

```diff
@@ -151,9 +143,9 @@
             <Bundle-Activator>io.stargate.auth.api.AuthApiActivator</Bundle-Activator>
             <Embed-Transitive>true</Embed-Transitive>
             <Import-Package><![CDATA[
-                            !org.slf4j,
-                            !org.slf4j.helpers,
-                            !org.slf4j.spi,
+                            org.slf4j,
```

Comment:
@dougwettlaufer do you remember why this explicitly excluded `org.slf4j` before?

Reply:
No but it's probably outdated now. As long as you can make a few requests to it and logs show up as expected then you should be good. I _think_ the big one is to make sure stacktraces show up because I vaguely remember there being some issue with those before.